### ISSUE TITLE
Support hubs

### ISSUE THREAD

**Issue by VIKTORVAV99** [matterdatabase/matterdatabase#9]
There are a few hubs that have added matter support to bridge their existing devices such as Philips Hue and IKEA Dirigera (in beta) and I'm sure there are several others as well.

It would be nice if this was reflected on the database.

**Comment by Pl8tinium** [comment]
Yeah and some devices that have been added in the matterdatabase and are available via hubs are currently not distinguishable between matter devices that dont require hubs (for setup and continous operation)

it would be great if we could filter this out as well. This is useful for people who dont want to buy hubs of different vendors

**Comment by matterdatabase** [comment]
The goal is to not show devices that are not matter native and require a hub (otherwise almost every ZigBee devices could be on the list)

Which devices that are on the list are not natively supported @Pl8tinium ?

**Comment by VIKTORVAV99** [comment]
> The goal is to not show devices that are not matter native and require a hub (otherwise almost every ZigBee devices could be on the list)
> 
> Which devices that are on the list are not natively supported [@Pl8tinium](https://github.com/Pl8tinium) ?

What I meant was to only show the hubs themselves that support it. Not all the devices that can connect to those hubs.

**Comment by Pl8tinium** [comment]
> The goal is to not show devices that are not matter native and require a hub (otherwise almost every ZigBee devices could be on the list)
> 
> Which devices that are on the list are not natively supported [@Pl8tinium](https://github.com/Pl8tinium) ?

i think for example the switchbot ultra lock

it requires their switchbot matter hub

<img width="1223" height="1152" alt="Image" src="https://github.com/user-attachments/assets/bdba5661-d51b-4941-b6bc-7b741b043d01" />

i dont have problems with these types of integrations, for some integrations i also keep them connected via a hub (eg rademacher roller shutters) as there are no valid alternatives in the space for them.

so i would still love to keep them listed but have a filter

regardless.. matterdatabase is a great contribution to the open smart home ecosystem. Thanks for maintaining this!

**Comment by Pl8tinium** [comment]
<img width="1039" height="211" alt="Image" src="https://github.com/user-attachments/assets/55b01fd9-338d-44ad-a8c1-69dbd6a4e19a" />

**Comment by Pl8tinium** [comment]
> > The goal is to not show devices that are not matter native and require a hub (otherwise almost every ZigBee devices could be on the list)
> > Which devices that are on the list are not natively supported [@Pl8tinium](https://github.com/Pl8tinium) ?
> 
> What I meant was to only show the hubs themselves that support it. Not all the devices that can connect to those hubs.

yup would also make sense, though listing all devices and just being able to filter for the integration type woudl be more flexible and atleast more useful for me :) (perhaps plus a "type" hub filter for you?)
